src/components/Tabs/Tab/Tab.tsx

```diff
@@ -1,14 +1,22 @@
 /* eslint-disable jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events */
+import cx from "classnames";
 import React, { FC, forwardRef, ReactElement, useRef } from "react";
 import { noop as NOOP } from "lodash-es";
-import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
 import Icon, { IconSubComponentProps } from "../../Icon/Icon";
 import VibeComponentProps from "../../../types/VibeComponentProps";
 import { IconType } from "../../Icon/IconConstants";
-import "./Tab.scss";
+import { ComponentDefaultTestId, getTestId } from "../../../tests/test-ids-utils";
+import styles from "./Tab.module.scss";
 
 export interface TabProps extends VibeComponentProps {
+  /**
+   * Class name for tab link-name
+   */
+  tabInnerClassName?: string;
```

Comment:
New prop added - to be able to remove overrides (from inside the package and from outside)

Reply:
So needed here https://github.com/DaPulse/dapulse/pull/53166

Reply:
@orrgottlieb @hadasfa 
Should it be done in another PR and released as a casual version?

Reply:
released as a casual version